stories/theme/core/RatingBar.css

```diff
@@ -1,18 +1,44 @@
+@value GoldStar: url("data:image/svg+xml;charset=utf8;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCIgdmlld0JveD0iMCAwIDUxIDQ4Ij48cGF0aCBmaWxsPSIjRkJCMTAwIiBkPSJNMjUgMWw2IDE3aDE4TDM1IDI5bDUgMTctMTUtMTAtMTUgMTAgNS0xN0wxIDE4aDE4eiIvPjwvc3ZnPg==";
+@value GrayStar: url("data:image/svg+xml;charset=utf8;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCIgdmlld0JveD0iMCAwIDUxIDQ4Ij48cGF0aCBmaWxsPSIjQ0VDRUNFIiBkPSJNMjUgMWw2IDE3aDE4TDM1IDI5bDUgMTctMTUtMTAtMTUgMTAgNS0xN0wxIDE4aDE4eiIvPjwvc3ZnPg==";
+
 .RatingBar .Label {
   text-align: center;
   display: inline;
   margin-left: 8px;
 }
 
-.RatingBar_Items {
-  display: inline-block;
+.RatingBar_Icons {
+  overflow: hidden;
+  height: 100%;
+  width: 0;
+  background-repeat: repeat-x;
 }
 
-.RatingBar_Item {
-  display: inline-block;
+.RatingBar_Background {
+  overflow: hidden;
+  max-width: 100px;
+  height: 20px;
+  width: 100px;
+  background-repeat: repeat-x;
```

Comment:
Since we are using `background-repeat` to fill bar with five stars, there are a lot of assuptions going on here:
* The width of the image is 20px
* We set the width to 100px (5 times 20px)

It would be good at least add some comments here:

```
width: 100px; /* Star width (20px) times 5 stars */
```